src/components/Navbar/index.module.scss

```diff
@@ -21,6 +21,8 @@
     }
   }
 
+  :global(:not(.ant-menu-dark).ant-menu-horizontal > .ant-menu-item),
```

Comment:
don't use `:not(.ant-menu-dark)` just put what the actual class name is you're trying to query